src/Illuminate/Database/Query/Builder.php

```diff
@@ -1338,20 +1337,18 @@ public function value($column)
      * Execute the query and get the first result.
      *
      * @param  array   $columns
-     * @return mixed|static
+     * @return \stdClass|array|null
```

Comment:
@GrahamCampbell you're getting the first item in the collection, which by default is an instance of `stdClass`. It could also be an array, if that's what you set `fetch` to.

If no results were returned, you get `null`.